<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QMainWindow" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Panel - MedPlus</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f7fa;
}
QTabWidget::pane {
    border: 1px solid #c0c0c0;
    background-color: white;
}
QTabBar::tab {
    background-color: #e0e0e0;
    padding: 8px 16px;
    margin-right: 2px;
}
QTabBar::tab:selected {
    background-color: white;
    border-bottom: 2px solid #2196f3;
}
QTableWidget {
    gridline-color: #e0e0e0;
    selection-background-color: #e3f2fd;
}
QHeaderView::section {
    background-color: #f0f0f0;
    padding: 8px;
    border: 1px solid #e0e0e0;
    font-weight: bold;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frameHeader">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblTitle">
         <property name="text">
          <string>&lt;h1&gt;üè• Admin Dashboard - MedPlus&lt;/h1&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogout">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    padding: 8px 16px;
    border: none;
    border-radius: 4px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #d32f2f;
}</string>
         </property>
         <property name="text">
          <string>üö™ Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabDashboard">
       <attribute name="title">
        <string>üìä Dashboard</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frameStats">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFrame" name="cardDoctors">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #42a5f5, stop:1 #1e88e5);
    border-radius: 8px;
    padding: 15px;
    color: white;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="lblCardIcon1">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 24px;
    background: transparent;
}</string>
                </property>
                <property name="text">
                 <string>üë®‚Äç‚öïÔ∏è</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblTotalDoctors">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 28px;
    font-weight: bold;
    background: transparent;
    color: white;
}</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 12px;
    background: transparent;
    color: rgba(255,255,255,0.8);
}</string>
                </property>
                <property name="text">
                 <string>Total Doctors</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="cardPatients">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #66bb6a, stop:1 #43a047);
    border-radius: 8px;
    padding: 15px;
    color: white;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="lblCardIcon2">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 24px;
    background: transparent;
}</string>
                </property>
                <property name="text">
                 <string>üë•</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblTotalPatients">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 28px;
    font-weight: bold;
    background: transparent;
    color: white;
}</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 12px;
    background: transparent;
    color: rgba(255,255,255,0.8);
}</string>
                </property>
                <property name="text">
                 <string>Total Patients</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="cardAppointments">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #ab47bc, stop:1 #8e24aa);
    border-radius: 8px;
    padding: 15px;
    color: white;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="lblCardIcon3">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 24px;
    background: transparent;
}</string>
                </property>
                <property name="text">
                 <string>üìÖ</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblTotalAppointments">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 28px;
    font-weight: bold;
    background: transparent;
    color: white;
}</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 12px;
    background: transparent;
    color: rgba(255,255,255,0.8);
}</string>
                </property>
                <property name="text">
                 <string>Total Appointments</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="cardRevenue">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #ff7043, stop:1 #f4511e);
    border-radius: 8px;
    padding: 15px;
    color: white;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="lblCardIcon4">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 24px;
    background: transparent;
}</string>
                </property>
                <property name="text">
                 <string>üí∞</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblTotalRevenue">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 28px;
    font-weight: bold;
    background: transparent;
    color: white;
}</string>
                </property>
                <property name="text">
                 <string>‚Çπ0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="styleSheet">
                 <string notr="true">QLabel {
    font-size: 12px;
    background: transparent;
    color: rgba(255,255,255,0.8);
}</string>
                </property>
                <property name="text">
                 <string>Total Revenue</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameRecent">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>&lt;h3&gt;üìã Recent Appointments&lt;/h3&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableRecentAppointments">
             <property name="styleSheet">
              <string notr="true">QMainWindow {
    background-color: #f5f7fa;
    color: #333333;
}

QWidget {
    background-color: #f5f7fa;
    color: #333333;
}

QTabWidget::pane {
    border: 1px solid #c0c0c0;
    background-color: white;
}

QTabBar::tab {
    background-color: #e0e0e0;
    padding: 8px 16px;
    margin-right: 2px;
    color: #333333;
}

QTabBar::tab:selected {
    background-color: white;
    border-bottom: 2px solid #2196f3;
    color: #333333;
}

QTableWidget {
    gridline-color: #e0e0e0;
    selection-background-color: #e3f2fd;
    background-color: white;
    alternate-background-color: #f8f9fa;
    color: #333333;
    border: 1px solid #e0e0e0;
    border-radius: 4px;
}

QTableWidget::item {
    padding: 8px;
    border-bottom: 1px solid #f0f0f0;
    color: #333333;
}

QTableWidget::item:selected {
    background-color: #e3f2fd;
    color: #333333;
}

QHeaderView::section {
    background-color: #f0f0f0;
    padding: 8px;
    border: 1px solid #e0e0e0;
    font-weight: bold;
    color: #333333;
}

QPushButton {
    background-color: #2196f3;
    color: white;
    padding: 8px 16px;
    border: none;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976d2;
}

QLineEdit {
    padding: 8px;
    border: 1px solid #e0e0e0;
    border-radius: 4px;
    background-color: white;
    color: #333333;
}

QFrame {
    background-color: white;
    color: #333333;
}</string>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameControls">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btnRefreshDashboard">
             <property name="text">
              <string>üîÑ Refresh Dashboard</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDoctors">
       <attribute name="title">
        <string>üë®‚Äç‚öïÔ∏è Doctors</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QFrame" name="frameDoctorControls">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="lineEditSearchDoctor">
             <property name="placeholderText">
              <string>üîç Search doctors...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnRefreshDoctors">
             <property name="text">
              <string>üîÑ Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnViewDoctorDetails">
             <property name="text">
              <string>üëÅÔ∏è View Details</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnToggleAvailability">
             <property name="text">
              <string>üîÑ Toggle Availability</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableDoctors">
          <property name="styleSheet">
           <string notr="true">QMainWindow {
    background-color: #f5f7fa;
    color: #333333;
}

QWidget {
    background-color: #f5f7fa;
    color: #333333;
}

QTabWidget::pane {
    border: 1px solid #c0c0c0;
    background-color: white;
}

QTabBar::tab {
    background-color: #e0e0e0;
    padding: 8px 16px;
    margin-right: 2px;
    color: #333333;
}

QTabBar::tab:selected {
    background-color: white;
    border-bottom: 2px solid #2196f3;
    color: #333333;
}

QTableWidget {
    gridline-color: #e0e0e0;
    selection-background-color: #e3f2fd;
    background-color: white;
    alternate-background-color: #f8f9fa;
    color: #333333;
    border: 1px solid #e0e0e0;
    border-radius: 4px;
}

QTableWidget::item {
    padding: 8px;
    border-bottom: 1px solid #f0f0f0;
    color: #333333;
}

QTableWidget::item:selected {
    background-color: #e3f2fd;
    color: #333333;
}

QHeaderView::section {
    background-color: #f0f0f0;
    padding: 8px;
    border: 1px solid #e0e0e0;
    font-weight: bold;
    color: #333333;
}

QPushButton {
    background-color: #2196f3;
    color: white;
    padding: 8px 16px;
    border: none;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976d2;
}

QLineEdit {
    padding: 8px;
    border: 1px solid #e0e0e0;
    border-radius: 4px;
    background-color: white;
    color: #333333;
}

QFrame {
    background-color: white;
    color: #333333;
}</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAppointments">
       <attribute name="title">
        <string>üìÖ Appointments</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QFrame" name="frameAppointmentControls">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="lineEditSearchAppointment">
             <property name="placeholderText">
              <string>üîç Search appointments...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnRefreshAppointments">
             <property name="text">
              <string>üîÑ Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnViewAppointmentDetails">
             <property name="text">
              <string>üëÅÔ∏è View Details</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableAppointments">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabReports">
       <attribute name="title">
        <string>üìä Reports</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QFrame" name="frameReportControls">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="btnRefreshStats">
             <property name="text">
              <string>üîÑ Refresh Stats</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnExportReport">
             <property name="text">
              <string>üì§ Export Report</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDetailedStats">
             <property name="text">
              <string>üìà Detailed Stats</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableDoctorStats">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
